docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc

```diff
@@ -0,0 +1,267 @@
+[[semantic-search-semantic-text]]
+=== Tutorial: semantic search with `semantic_text`
+++++
+<titleabbrev>Semantic search with `semantic_text`</titleabbrev>
+++++
+
+beta[]
+
+This tutorial shows you how to use the semantic text feature to perform semantic search on your data.
+
+Semantic text simplifies the {infer} workflow by providing {infer} at ingestion time and sensible default values automatically.
+You don't need to define certain settings and parameters or create {infer} ingest pipelines.
+
+The recommended way to use semantic search in the {stack} is following the semantic text workflow.
+When you need more control over the settings, you can still use the complete {infer} workflow (refer to  <<semantic-search-inference,this tutorial>> to review the process).
+
+This tutorial uses the `elser` service for demonstration but you can use any service and their supported models offered by the {infer-cap} API.
```

Comment:
Link to ELSER docs?

Reply:
Added via [6a086bc](https://github.com/elastic/elasticsearch/pull/108311/commits/6a086bc02815060ae56a3320aea9e0ab23d413c9).